modules/context/repo.go

```diff
@@ -525,7 +525,7 @@ func getRefName(ctx *Context, pathType RepoRefType) string {
 		return getRefNameFromPath(ctx, path, ctx.Repo.GitRepo.IsTagExist)
 	case RepoRefCommit:
 		parts := strings.Split(path, "/")
-		if len(parts) > 0 && len(parts[0]) == 40 {
+		if len(parts) > 0 && len(parts[0]) >=7 && len(parts[0]) <= 40 {
```

Comment:
Issues with gofmt, the below suggestion should pass on CI
```suggestion
		if len(parts) > 0 && len(parts[0]) >= 7 && len(parts[0]) <= 40 {
```

Reply:
Done, thanks!